README.md

```diff
@@ -186,6 +186,15 @@ Here is a list of the commands I use when developing Porcupine:
     See [#472](https://github.com/Akuli/porcupine/issues/472).
 - `python3 -m pytest` runs tests. You will see lots of weird stuff happening
   while testing, and that's expected.
+    A good way to debug a test to see what is actually going on is to add traces.
+    It pauses the test to show you the current state of the program.
+      - Use `import pdb` to implement the functionality
+      - Use `pdb.set_trace()` to set the pause points in the program. You can
+        set as many as you like. (A good way is to use `import pdb; pdb.set_trace()`
+        for the first pause and `pdb.set_trace()` for successive pauses)
```

Comment:
I don't like "implement the functionality". How about this? Also changed "program" --> "test". You can put them anywhere, but they seem to be the most useful in tests.

```suggestion
      - Use `import pdb` and `pdb.set_trace()` to set the pause points in the test. You can
        set as many as you like, and it can conveniently be done on one line: `import pdb; pdb.set_trace()`.
```